src/api/routes/core-node-rpc-proxy.ts

```diff
@@ -21,6 +21,38 @@ function getReqUrl(req: { url: string; hostname: string }): URL {
   return new URL(req.url, `http://${req.hostname}`);
 }
 
+// https://github.com/stacks-network/stacks-core/blob/20d5137438c7d169ea97dd2b6a4d51b8374a4751/stackslib/src/chainstate/stacks/db/blocks.rs#L338
+const MINIMUM_TX_FEE_RATE_PER_BYTE = 1;
+
+function getFeeEstimationModifier(): number | null {
```

Comment:
Can we do this in a lazy way? e.g. parse once and keep it cached in a global var

Reply:
Let me see if I can do that in a way that still lets the test configure it..

Reply:
Done